packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,27 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
-
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+## Troubleshooting
+
+### How to make Netlify forms work?
+
+1. Add the following [prerendering configuration](https://kit.svelte.dev/docs#configuration-prerender) to `svelte.config.cjs`:
```

Comment:
`cjs` was only used in old versions. If you're on the latest version of SvelteKit it will be `js`

```suggestion
1. Add the following [prerendering configuration](https://kit.svelte.dev/docs#configuration-prerender) to `svelte.config.js`:
```

Reply:
Ah, it looks like that might have been taken directly from the article. @sw-yx maybe you'd want to update that